platform/src/main/java/platform/pageobjects/MyPage/ProductDetailsFilterValidation.java

```diff
@@ -0,0 +1,253 @@
+				if (productValue.contains((kanjikenteiProduct))) {
+					Select productselect = new Select(productDropdown);
+					productselect.selectByVisibleText(kanjikenteiProduct);
+					log.info("Free user is Selected Kanji Kaneti product");
+					narrowDownButton.click();
+					List<WebElement> pagination = driver.findElements(By.xpath("//div[@class='pagination']/div"));
+					log.info("Total Pages " + pagination.size());
+					int size = pagination.size();
+					if (size > 0) {
+						for (int k = 1; k <= pagination.size(); k++) {
+							driver.findElement(By.xpath("//div[@class='pagination']/div[" + k + "]")).click();
+							List<WebElement> elements = driver
+									.findElements(By.xpath("//div[@class='ep-product-list__row']/div"));
+							log.info("Total cards in Page " + elements.size());
+							for (int j = 0; j < elements.size(); j++) {
+								if ((elements.get(j).getText()).contains(kanjikenteiProduct)) {
+									log.info("This is Kanji Kaneti product card " + j + " Of " + elements.size()
+											+ " Total Cards");
+								}
+							}
+							// break;
+
+						}
+
+					}
+				}
+			}
+		} else {
+			log.info("pagination not exists");
+		}
+
+	}
+
+	public void CheckPrimeMembership_CASEC_Productetails() throws Exception {
+		if (CommonFunctions.waitForVisiblity(productDropdown, waitTime)) {
+//			ExcelUtil excel = new ExcelUtil();
+//			excel.setExcelFile("NewTestData.xlsx", "MembershipStatus");
+			casecProduct = "CASEC";
+			Select target = new Select(productDropdown);
+			List<WebElement> targetListElements = target.getOptions();
+			int DropdownitemSize = targetListElements.size();
+			for (int i = 0; i < DropdownitemSize; i++) {
+				String productValue = targetListElements.get(i).getText();
+				if (productValue.contains((casecProduct))) {
+					Select productselect = new Select(productDropdown);
+					productselect.selectByVisibleText(casecProduct);
+					log.info("Prime user is Selected CASEC product");
+					narrowDownButton.click();
+					List<WebElement> pagination = driver.findElements(By.xpath("//div[@class='pagination']/div"));
+					log.info("Total Pages " + pagination.size());
+					int size = pagination.size();
+					if (size > 0) {
+						for (int k = 1; k <= pagination.size(); k++) {
+							driver.findElement(By.xpath("//div[@class='pagination']/div[" + k + "]")).click();
+							List<WebElement> elements = driver
+									.findElements(By.xpath("//div[@class='ep-product-list__row']/div"));
+							log.info("Total cards in Page " + elements.size());
+							for (int j = 0; j < elements.size(); j++) {
+								if ((elements.get(j).getText()).contains(casecProduct)) {
+									log.info("This is CASEC product card " + j + " Of " + elements.size()
+											+ " Total Cards");
+								}
+							}
+
+						}
+						break;
+
+					}
+				}
+			}
+		} else {
+			log.info("pagination not exists");
+		}
+
+	}
+
+	public void CheckPrimeMembership_kanjikentei_Productetails() throws Exception {
```

Comment:
Please remove _ (underscore) and spell chk

Reply:
done